addons/packages/antrea/1.2.3/bundle/config/values.yaml

```diff
@@ -6,6 +6,7 @@ infraProvider: vsphere
 antrea:
   config:
     serviceCIDR: 10.96.0.0/12
+    serviceCIDRv6: null
```

Comment:
These are not required when AntreaProxy is enabled, which seems to be the same (see `featureGates` below)

https://github.com/antrea-io/antrea/blob/9a52de90e90a8beeb512f56c51b670d93ec82325/build/yamls/antrea.yml#L4050-L4059

Reply:
In the overlay we currently omit the `serviceCIDRv6` if it is null or unset. [On line 103 of the overlay.](https://github.com/tylerschultz/community-edition/blob/5736884161ab0fd127b9adac5524380b8c0fd5e0/addons/packages/antrea/1.2.3/bundle/config/overlay/antrea_overlay.yaml#L103)

We could add a check to see if the `featureGate` for `AntreaProxy` is true, to avoid setting the `serviceCIDRv6` in that case. Would that be the best thing to do? Is there a risk in setting the `serviceCIDRv6` if the `AntreaProxy` flag is true?

Reply:
there is no risk as the field is ignored (there isn't even a warning in the logs IIRC) when AntreaProxy is enabled. so no issue with always emitting these CIDRs, if there is a possibility for the user to disable AntreaProxy.

Reply:
The users can set any of the values in the `values.yaml`, so they could set the `AntreaProxy` feature to `false`. We'll leave it as it is in this case.